src/libraries/System.Memory/tests/ArrayBufferWriter/ArrayBufferWriterTests.T.cs

```diff
@@ -205,6 +205,19 @@ public void GetMemory_DefaultCtor(int sizeHint)
             Assert.Equal(sizeHint <= 256 ? 256 : sizeHint, memory.Length);
         }
 
+        // NOTE: GetMemory_ExceedMaximumBufferSize test is constrained to run on Windows and MacOSX because it causes 
+        //       problems on Linux due to the way deferred memory allocation works. On Linux, the allocation can 
+        //       succeed even if there is not enough memory but then the test may get killed by the OOM killer at the 
+        //       time the memory is accessed which triggers the full memory allocation. 
+        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.OSX)]
+        [ConditionalFact(nameof(IsX64))]
+        [OuterLoop]
+        public void GetMemory_ExceedMaximumBufferSize()
```

Comment:
Does this test actually end up allocating a lot or taking a while? If it isn't resource intensive (which it doesn't look like to be), it could just be `[Fact]`.

Reply:
We may also remove the OuterLoop?

Reply:
Yes.